source/src/microapp/cs_Microapp.cpp

```diff
@@ -104,6 +109,14 @@ void Microapp::updateStateFromOperatingData(uint8_t index) {
 	_states[index].didReboot         = (prevState == MicroappOperatingState::CS_MICROAPP_RUNNING);
 }
 
+void Microapp::onExcessiveCallDuration(uint8_t appIndex) {
+	if (appIndex < g_MICROAPP_COUNT && _states[appIndex].exceededCallDuration == false) {
```

Comment:
Inverted boolean check? I expect that `.exceededCallDuration == true` is the fault-state.

Reply:
Means it was false, and now becomes true.